pkgs/development/python-modules/blist/default.nix

```diff
@@ -14,6 +15,13 @@ buildPythonPackage rec {
     sha256 = "1hqz9pqbwx0czvq9bjdqjqh5bwfksva1is0anfazig81n18c84is";
   };
 
+  patches = [
+    (fetchpatch {
```

Comment:
can you add comment describing why this is necessary?

Reply:
resolved